src/testRunner/unittests/tsserver/projects.ts

```diff
@@ -1620,5 +1609,40 @@ namespace ts.projectSystem {
 
             checkNumberOfInferredProjects(projectService, 0);
         });
+
+        it("file opened is in configured project that will be removed", () => {
+            const testsConfig: File = {
+                path: `${tscWatch.projectRoot}/playground/tsconfig.json`,
+                content: "{}"
+            };
+            const testsFile: File = {
+                path: `${tscWatch.projectRoot}/playground/tests.ts`,
+                content: `export function foo() {}`
+            };

```

Comment:
This is a bit of a nit - but really, is the idea that that `playground` contains some sort of implementation file, and `playground/tests` would contain the test file? In theory, you should only need 2 tsconfigs and 2 `.ts` files - but you don't need `innerSrcFile`, right?

```
src/
├── tsconfig.json
├── foo.ts
└── spec/
    ├── tsconfig.json
    └── foo.ts
```